### PR TITLE
update kernel install instructions.

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

**Comment by minrk** [comment]
Want to PR against #8877, which has much of the same info?

**Comment by Carreau** [comment]
> Want to PR against #8877, which has much of the same info?

rebased.

File: docs/source/install/kernel_install.rst [hunk 1]
```diff
@@ -11,10 +11,15 @@ environments) as a kernel for Jupyter by following these steps:
   and ipykernel is installed
 * run once ``ipython kernel install --user``,
   or ``python -m ipykernel install --user`` to ensure a specific Python installation is used.
+* See `ipython kernel install --help` for the list of installation options like
+  naming the kernel, or non default install location.
+* The IPython kernel for Jupyter is provided by the `ipykernel` python package,
+  see there if you need more flexibility for installation.
```
Comment (damianavila) [inline docs/source/install/kernel_install.rst]: `there`, probably a link should be nice here...

File: docs/source/install/kernel_install.rst [hunk 2]
```diff
@@ -11,10 +11,15 @@ environments) as a kernel for Jupyter by following these steps:
...
 
 
-The last command installs a :ref:`kernel spec <jupyterclient:kernelspecs>` file for the current python installation. Kernel spec files are JSON files, which can be viewed and changed with a
-normal text editor.
+The last command installs a :ref:`kernel spec <jupyterclient:kernelspecs>` file
```
Comment (damianavila) [inline docs/source/install/kernel_install.rst]: Probably this sentence is better placed after the examples, not before them...

**Comment by Carreau** [comment]
closing in favor of #9008

**Pull request closed without merging**